pkg/kn/commands/revision/revision.go

```diff
@@ -29,3 +31,26 @@ func NewRevisionCommand(p *commands.KnParams) *cobra.Command {
 	revisionCmd.AddCommand(NewRevisionDeleteCommand(p))
 	return revisionCmd
 }
+
+// ============================================
+// Shared revision functions:
+
+// Extract traffic and tags for given revision from a service
+func trafficAndTagsForRevision(revision string, service *v1alpha1.Service) (int64, []string) {
+	if len(service.Status.Traffic) == 0 {
+		return 0, nil
+	}
+	var percent int64
+	var tags []string
+	for _, target := range service.Status.Traffic {
+		if target.RevisionName == revision {
+			if target.Percent != nil {
+				percent += *target.Percent
+			}
+			if target.Tag != "" {
+				tags = append(tags, target.Tag)
```

Comment:
For a revision tagged twice in targets: revision list will show the sum of traffic this revision is receiving and comma separated list of tags assigned. Should we list comma separated traffic portions as we're doing here for tags ?

Reply:
Well, I think this is an information that is not so important for a list overview entry. Important is the total amount of traffic targeted to this list. I'm fine with the current view.

Reply:
> that is not so important for a list overview entry.

+1